src/tutorials/0005-regular-files-api/04.md

```diff
@@ -17,6 +17,8 @@ The `cat` method returns an Async Iterable that iterates over the file's chunks
 
 A `Buffer` is just a raw collection of bytes and as such doesn't make any assumptions about encodings or the type of data it contains. However, if we know the file being retrieved is a plain text file such as a `.txt`, we can convert its buffered contents to a UTF-8 string (an interpretation of those raw bytes) by calling the JavaScript method `.toString()`.
 
+But since we have multiple buffers of the same file, we need to reassemble them (concatenate) into a single buffer before converting it into a string. The [`it-concat`](https://www.npmjs.com/package/it-concat) package can do just that: iterate over all of the chunks of the file and put them back together for us.
```

Comment:
```suggestion
But since we have multiple buffers making up a single file, we need to reassemble them (concatenate) into a single buffer before converting it into a string. The [`it-concat`](https://www.npmjs.com/package/it-concat) package can do just that: iterate over all of the chunks of the file and put them back together for us.
```

Reply:
FWIW `it-concat` turns the buffers into a [`BufferList`](https://www.npmjs.com/package/bl), which you have to call `.slice` on to get an actual `Buffer`.

It's a good approach if you are going to doing lots of operations on the contents afterwards or appending new data, etc but if you want to avoid the conceptual overhead of Buffers vs BufferLists, you can just use [`it-to-buffer`](https://www.npmjs.com/package/it-to-buffer) instead.

Reply:
hey @achingbrain! Thank you for the input.

That's a good point, we'll switch to `it-to-buffer` then 👍 
Also, I just noticed that `it-concat` depends on [`bl`](https://www.npmjs.com/package/bl), so maybe it is intended for nodejs and not browers? @alanshaw 
If that's the case, then it's one more reason to switch to `it-to-buffer`.